docs/src/main/sphinx/routines/examples.md

```diff
@@ -80,6 +80,26 @@ SQL routine invocation works without the full path:
 SELECT answer() + 5; -- 47
 ```
 
+You can show SQL routines and plugin functions in a schema. This example uses
+the `default` schema of the `example` catalog:
+
+```sql
+SHOW FUNCTIONS FROM example.default
+```
+
+Of the functions returned, find all that match a character pattern. This example
+returns results with a name beginning with `ans`:
+
+```sql
+SHOW FUNCTIONS like 'ans%';
+```
+
+Show the SQL statement that created the function. In this example the function is `answer`:
```

Comment:
Add note that the () is not used and that if there are multiple routines with such a name but different parameters they are all returned.